electron/launcher/main-process/ipc-listeners/user-self-help.js

```diff
@@ -0,0 +1,42 @@
+const request = require('request-promise');
```

Comment:
Use the request wrapper instead of request by itself. It adds CSRF and Session hash to the HTTP headers.
`const request = require('helpers/request-wrapper').request;`

Reply:
Done.